scripts/reset_wall.sh

```
@@ -4,22 +4,22 @@
+IMAGE="$2"
 
 # Input validation
 if [ -z $VOLUME ]; then
-    echo "Usage: $0 <pg_volume> [<pg_version>]"
+    echo "Usage: $0 <pg_volume> [<pg_image>]"
     exit 1
 fi
 
-if [ -z $VERSION ]; then
- VERSION="latest"
+if [ -z $IMAGE ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)